samples/ibm-samples/openscale/README.md

```diff
@@ -0,0 +1,116 @@
+# Watson OpenScale Example
+
+This simple OpenScale pipeline will demonstrate how to train a model using IBM Spark Service, then deploy it with Watson Machine Learning, and subscribe the model with Watson OpenScale monitoring.
```

Comment:
This simple OpenScale pipeline will demonstrate how to train a model using IBM Spark Service, store and deploy it with Watson Machine Learning, and then Watson OpenScale for fairness and quality monitoring

Reply:
done